connect/runtime/src/main/java/org/apache/kafka/connect/runtime/errors/RetryWithToleranceOperator.java

```diff
@@ -331,6 +351,15 @@ public synchronized Throwable error() {
         return this.context.error();
     }
 
+    /**
+     * Exit from any currently ongoing retry loop and mark the operation as failed.
+     * This can be called from a separate thread to break out of an infinite retry loop in
```

Comment:
Nit: this is useful for more than just infinite retries
```suggestion
     * This can be called from a separate thread to break out of retry/backoff loops in
```